### ISSUE TITLE
[A11y_Bug]: Issue type counts should not be in tab order / clickable

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1819]
### Project

extension-checker

### Browser

Chrome

### Operating System

MacOS

### Assistive technology

Keyboard

### Description

The keyboard tab stops on the issue type/icon counts, 
as if to say they are interactive, but they are not.
Pressing enter/space only shows the tooltip, confusing the keyboard user that there is some other action available because the tab stopped there.  The tooltip should show when t receives keyboard focus, without needing to pres space/enter to show it.  However, in this case, they shouldn't be in the tab order at all.
![IssueTypeCounts](https://github.com/IBMa/equal-access/assets/16193609/2573d2c8-63ba-464a-9089-07c729c9cad2)

### Steps to reproduce

1. Open Checker
2. Tab until focus arrives on 'issue type/icons' with counts

**Comment by marcjohlic** [comment]
Discussion with Marc, Tom, Mike, Josiah & Marnie - decision is to just remove the Definition ToolTip completely from these icons as we already have affordances for these in the QuickGuide, User Guide, and Issues Panel.

Action:  Remove Definition ToolTip from these icons.

**Comment by philljenkins** [comment]
@drjoho 
JAWS 2024 currently reads the controls as follows, which is not correct.
Not using the `tab key` to navigate, but using the `right arrow keys` to reads the UI:
after scanning AltoroMutual 
...
- "Keyboard Checker Mode button - to activate press enter"
- "Violation button collapsed - to activate press enter"
- pressing `Enter`, JAWS says "expanded" and nothing else
- continuing with `right arrow key`, JAWS will read the numbers one at a time, and then spell out "Violations" one letter at a time.
- continuing instead with `down arrow key`, JAWS will correctly read the "blank", then "Violations" as a word, not character by character, but then reads (incorrectly) the number followed by a bunch of other numbers, control names, and states (collapsed).

**Comment by drjoho** [comment]
Deemed fixed in new UI
